lighthouse-core/test/audits/users-rel-preconnect-test.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+/* eslint-env mocha */
+
+const UsesRelPreconnect = require('../../audits/uses-rel-preconnect.js');
+const assert = require('assert');
+const Util = require('../../report/v2/renderer/util');
+
+const mainResource = {
+  url: 'https://www.example.com/',
+  _endTime: 1,
+};
+const mainResourceRecord = {
```

Comment:
why do these two need to be different? :)

Reply:
good question 😛